lib/fs.js

```diff
@@ -1927,11 +1927,8 @@ Object.defineProperties(fs, {
     configurable: true,
     enumerable: false,
     get() {
-      if (promises === null) {
```

Comment:
Please make this property `enumerable: true` now that it's no longer experimental.

Reply:
It also no longer needs to be a getter.

Reply:
@jdalton Thanks, done :)